ansible/inventory/group_vars/defaults.yaml

```diff
@@ -32,6 +32,12 @@ aws:
   tags:
     - Key: overwrite-me
       Value: overwrite-me
+  certificate_arn: overwrite-me
+  cert_region: ap-southeast-2
+  certificate_key_arn: overwrite-me
+  cert_key_region: ap-southeast-2
+
+cert_tmp_dir: /tmp/shinesolutions/packer-aem/certs
```

Comment:
should this be hard coded?

Reply:
that `cert_tmp_dir` is a default configuration value... so it's configurable and the user can overwrite the value....